akka-http-avro4s/src/main/scala/de/heikoseeberger/akkahttpavro4s/AvroSupport.scala

```diff
@@ -81,13 +81,17 @@ trait AvroSupport {
         try {
           output.write(data)
         } finally {
-          // Ignoring closing exception
-          Try(output.close())
+          try output.close()
+          catch {
+            case _: IOException => // Ignoring closing exceptions
```

Comment:
Thanks for the change. Now I have a question: is it safe to ignore the exception? I don't know Avro, so I might be totally wrong, but I could imagine `output.write` to just buffer and `output.close` to actually write, in which case the exception should not be ignored.